app/uploaders/avatar_uploader.rb

```diff
@@ -7,19 +7,6 @@ class AvatarUploader < CarrierWave::Uploader::Base
   # Choose what kind of storage to use for this uploader:
   storage :fog
 
-  # Override the directory where uploaded files will be stored.
-  # This is a sensible default for uploaders that are meant to be mounted:
-  
-
-  def store_dir
-    'images'
```

Comment:
This will definitely break production.  You should put `store_dir` and `cache_dir` back as they were.